include/grpc/grpc.h

```diff
@@ -533,8 +533,27 @@ GRPCAPI char* grpc_channelz_get_subchannel(intptr_t subchannel_id);
    is allocated and must be freed by the application. */
 GRPCAPI char* grpc_channelz_get_socket(intptr_t socket_id);
 
+/************* Shutdown support. *************/
+/* Register a function to be called at the end of the last grpc_shutdown() call. */
+GRPCAPI void grpc_on_shutdown_callback(void (*func)());
```

Comment:
Any changes to the API need a gRFC (a PR in grpc/proposal)

Reply:
Thanks, I will do that.